src/EmailTemplate.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Beaver\Dateval;
+
+class EmailTemplate {
+    public $path;
+    
+    private $templateExtenstion = ".html";
+    private $template;
+    
+    public $parsedTemplate;
+    
+    public function __construct($path) {
+        $path .= $this->templateExtenstion;
+        
+        if(!file_exists($path)) {
+            throw new PathException("Provided email template doesn't exist on that location");
+            
+        } else {
```

Comment:
__[phpmd]__
* The method __construct uses an else expression. Else is never necessary and you can simplify the code to work without else.